en/rules.md

```diff
@@ -86,16 +86,16 @@ Its Strength cannot exceed 66.
 *Slider AA*: 1 or below.
 
 ##### Timewarp:
-As Previously Mentioned: your Timewarp speed must be at minimum 60% of 1.5x speed for Double Time*, and 60% of 1x speed for everything else.
+As Previously Mentioned: your Timewarp speed must be at minimum 60% of 1.5x speed for Double Time*, and 1x speed for everything else.
 
 For AQN, That's:
-66 **Without DT**, 100 **With DT**.
+100 **Without DT**, 100 **With DT**.
 
 For osu!rx, That's:
-0.6 **Without DT**, 1.0 **With DT**.
+1.0 **Without DT**, 1.0 **With DT**.
 
 For Maple That's:
```

Comment:
i'd say to merge maple and aqn because they are the same thing (`For Maple/AQN, That's:`)